Exosite.cpp

```diff
@@ -1938,17 +1938,3 @@ boolean Exosite::fetchNVCIK(){
     return false;
   }
 }
-
-
-
-boolean Exosite::isHex(char *str, int len){
```

Comment:
Leave this code in place, just don't call it. Leave a comment on why this changed, why we're no longer calling, and why we're leaving it in place (i.e. it might get reverted).